okio/src/jvmMain/kotlin/okio/internal/zip.kt

```diff
@@ -55,10 +55,17 @@ private const val HEADER_ID_EXTENDED_TIMESTAMP = 0x5455
 /**
  * Opens the file at [zipPath] for use as a file system. This uses UTF-8 to comments and names in
  * the zip file.
+ *
+ * @param predicate a function that returns false for entries that should be omitted from the file
```

Comment:
TIOLI: a filter for entries...

Reply:
I always always need to check whether true means keep or true means discard. Saying exactly that is my hope at helping at least myself.